### PR TITLE
5 pay money to the artist

### LINKED ISSUE #5: 5-pay-money-to-the-artist
As a user,
So that i can support the artists whose music I enjoy,
I would like to be able to pay money to the artist.

### PR DESCRIPTION
**Pull request by ghost**
closes #5

### CODE REVIEW

File: back-end/app/controllers/streams_controller.rb [hunk 1]
```diff
@@ -1,27 +1,42 @@
-require "assets/stream_factory.rb"
 class StreamsController < ApplicationController
-  
+
   FIRST_PLAY=1
 
   def create
+    user_id = params[:user_id] ? params[:user_id] : 1000
```
Comment (MarcoBgn) [inline back-end/app/controllers/streams_controller.rb]: change user number back to 1

File: back-end/app/controllers/streams_controller.rb [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
+    stream.weekly_plays + 1
   end
-  
-  def user_id
-    current_user ? current_user.id : 1
-  end
-end
\ No newline at end of file
+
+  # def user_id
```
Comment (MarcoBgn) [inline back-end/app/controllers/streams_controller.rb]: delete commented code

File: front-end/app/scripts/controllers/login.js [hunk 3]
```diff
@@ -9,14 +9,17 @@ angular.module('fairMuseApp')
...
-                    password: self.password};
-       var promise = authenticationService.login(self.user);
+      self.user = {email: self.email,
+                   password: self.password};
+      var promise = authenticationService.login(self.user);
       promise.then(success, error);
     };
 
       var success = function(response){
+        console.log(response.data)
```
Comment (MarcoBgn) [inline front-end/app/scripts/controllers/login.js]: remove console.log

File: front-end/app/scripts/controllers/user.js [hunk 4]
```diff
@@ -2,4 +2,27 @@
...
+          var promise = self.getStreamDetails();
+          promise.then(success, error);
+        };
+
+    self.getStreamDetails = function(){
+      return $http.post('http://localhost:3000/api/streaminfo', {user_id: localStorage.getItem('user_id')})
+  };
+
+  var success = function(response){
+    console.log(response.data);
```
Comment (MarcoBgn) [inline front-end/app/scripts/controllers/user.js]: remove console.log

File: front-end/app/views/navbar.html [hunk 5]
```diff
@@ -17,12 +17,17 @@
         <ul class="nav navbar-nav">
           <li class="active"><a href="#/" >Home</a></li>
           <li><a ng-href="#/about">About</a></li>
-          <li><a ng-href="#/artist">Artists</a></li>
-          <li><a ng-href="#/artistsignin">Artist Sign In</a></li>
+          <!-- <button ng-click="index.clearLocalStorage()">Clear</button> -->
```
Comment (MarcoBgn) [inline front-end/app/views/navbar.html]: delete comments

**Comment by yanyil** [comment]
:shipit:

**Pull request merged**